### ISSUE TITLE
Questions.yaml does not show subquestions with different input values at show_subquestion_if for the same variable.

### ISSUE THREAD

**Issue by bhagirathhapse** [rancher/partner-charts#334]
In my product, `TrilioVault for Kubernetes`, we are trying to insert the value for a variable as `true | false`, and depending on the user input we want to show different subquestions to the user and get more inputs.
What I observed is that `questions.yaml` is not able to show_subquestions when the same variable has different input values and then shows different subquestions depending on the input value. Refer the questions.yaml file [here](https://github.com/bhagirathhapse/partner-charts/blob/main-source/charts/k8s-triliovault-operator/k8s-triliovault-operator/2.6.704/questions.yaml)

In the above file, to make the variable `installTVK.ComponentConfiguration.ingressController.enabled` work, I have to add the same variable twice with `true` and `false` values.
Let me know if there is a better way to manage this sceanario.

**Comment by bhagirathhapse** [comment]
I have discussed this issue with @samuelattwood and he helped me with a workaround (I guess) and it worked for my use-case.
We have used `show_if` parameter type to resolve the issue and then I could list the new questions for different values of `installTVK.ComponentConfiguration.ingressController.enabled`.

I am using below lines to add different questions for different input values from the users.

`show_if: "installTVK.ComponentConfiguration.ingressController.enabled=true"`
and 
`show_if: "installTVK.ComponentConfiguration.ingressController.enabled=false"`
